src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -325,7 +312,7 @@ internal sealed class MemberSignatureComparer : IEqualityComparer<Symbol>
             considerReturnType: true,
             considerTypeConstraints: false,
             considerCallingConvention: true,
-            considerRefKindDifferences: true,
+            refKindCompareMode: RefKindCompareMode.ConsiderDifferences,
```

Comment:
Consider adding a test showing that `RetargetedExplicitImplementationComparer` remains strict

Reply:
Haven't figured out how to observe the difference for `RetargetedExplicitImplementationComparer` - it's used to find the method in the retargeted type, but that method seems to always have the correct signature (since it has just been retargeted). And I don't think `RecordAPISignatureComparer` can be tested with the new mismatches (since there's no synthesized API that would have `ref` or `in` parameter).

Reply:
Actually with Aleksey's help managed to test the retargeting comparer. See [https://github.com/dotnet/roslyn/pull/69205](https://github.com/dotnet/roslyn/pull/69205)